lib/modules/dosomething/dosomething_mbp/dosomething_mbp.module

```diff
@@ -105,42 +105,43 @@ function dosomething_mbp_get_transactional_payload($origin, $params = NULL) {
   // Payload specific to each transaction type.
   switch ($origin) {
     case 'user_register':
+      $payload['template'] = variable_get('dosomething_mbp_user_register_template');
       $payload['birthdate'] = $params['birthdate'];
       $payload['subscribed'] = 1;
       if (isset($params['mobile'])) {
         $payload['mobile'] = $params['mobile'];
       }
       $payload['merge_vars']['FNAME'] = $params['first_name'];
-      $payload['email_template'] = 'mb-general-signup';
```

Comment:
`dosomething_mbp_update_7004()` will take care of setting the necessary variables based on the country code of the site - US and each of the affiliate sites.

Tested with a clean staging DB resulting in the expected template settings which would be different for each site depending on the country code system variable. 

Could use a test run with an affiliate staging db. Didn't have any luck with `ds pull stage.uk --db` -> `ERROR 1049 (42000): Unknown database 'dosomething_uk'`.